elytron-oidc-client/subsystem/src/main/resources/schema/wildfly-elytron-oidc-client_1_0.xsd

```diff
@@ -0,0 +1,178 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema"
```

Comment:
Probably as a follow up activity it may be nice to get some documentation into the schema.

Reply:
+1, am planning to add that.